docs/ttl.md

```diff
@@ -79,4 +79,4 @@ The TransIP Provider minimal TTL is used when the TTL is 0. The minimal TTL is 6
 The Vultr provider minimal TTL is used when the TTL is 0. The default is 1 hour.
 
 ### UltraDNS 
-The UltraDNS provider minimal TTL is used when the TTL is 0. The default is 24 hour.
+The UltraDNS provider minimal TTL is used when the TTL is not given. The default ttl is account level ttl and if account level ttl is not defined then it is 24 hour.
```

Comment:
The UltraDNS provider minimal TTL is used when the TTL is not provided. The default TTL is account level default TTL, if defined, otherwise 24 hours.